backend/src/sms/services/sms.service.ts

```diff
@@ -65,6 +68,7 @@ const getHydratedMessage = async (
  * @param credential
  * @param recipient
  * @param message
+ * @param campaignId
```

Comment:
this seems unnecessary? (actually all these JSDocs stuff should be removed since we have TypeScript...but that's for another day)

Reply:
removed